Doc/library/struct.rst

```diff
@@ -405,6 +405,12 @@ The :mod:`struct` module also defines the following type:
    methods is more efficient than calling the :mod:`struct` functions with the
    same format since the format string only needs to be compiled once.
 
+   .. note::
+
+      The compiled versions of the most-recent format strings passed to
+      :class:`Struct` objects and the module-level functions are cached, so
```

Comment:
Do you mean Struct constructors? Once you have a Struct object, I don’t think you pass a format string to it in normal usage.